docs/diagrams/ModelClassDiagram.puml

```diff
@@ -5,11 +5,11 @@ skinparam arrowColor MODEL_COLOR
 skinparam classBackgroundColor MODEL_COLOR
 
 Package Model <<Rectangle>>{
-Interface ReadOnlyAddressBook <<Interface>>
+Interface ReadOnlyPawbook <<Interface>>
```

Comment:
If you look at the source code, `ReadOnlyPawbook` doesn't exist, those references to `AddressBook` has been renamed to `Database` instead.